blocks/about/__text/about__text.css

```diff
@@ -0,0 +1,29 @@
+.about__text {
+  font-size: 20px;
+  line-height: 28px;
+  margin: 14px 0 34px;
+}
+
+.about__text:nth-of-type(2) {
```

Comment:
Usually, it's a bad practice to use nth selector. How it will work when `about` blocks contain 3 text elements?

Reply:
Done